### PR TITLE
(DOCSP-27294): Fix typo and missing optional UUID

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

- https://jira.mongodb.org/browse/DOCSP-27294

### Staged Changes

- [Supported Types](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-27294/)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/swift/model-data/define-model/supported-types.txt [hunk 1]
```diff
@@ -397,7 +397,10 @@ Property Cheat Sheet
                 :copyable: false
                 
                 @property NSUUID *uuid;
-           - 
+           - .. code-block:: objectivec
+                :copyable: false
+                
+                @property NSUUID *uuid;
```
Comment (mongodben) [inline source/sdk/swift/model-data/define-model/supported-types.txt:403]:
```suggestion
                @property NSUUID *uuid;

```

Comment (mongodben) [reply source/sdk/swift/model-data/define-model/supported-types.txt:403]: nit: dont think this matters for snooty, but keeps formatting consistent

**Review by mongodben — APPROVED** [review]
lgtm w 1 nit

File: source/sdk/swift/model-data/define-model/supported-types.txt (hunk 1 above)
Comment (dacharyc) [reply source/sdk/swift/model-data/define-model/supported-types.txt:403]: Actually, adding an empty line after 403 would make the formatting inconsistent with the rest of the table. I understand the instinct, though!

**Pull request merged**